### PR TITLE
feat: top-bar shortcut to today's journal daily summary (#876)

### LINKED ISSUE #876: feat: top-bar shortcut to today's (or most recent) journal daily summary
## Summary

Add an icon button to the top-bar icon cluster (next to the notification bell + settings) that opens today's journal daily summary in FilesView. When today's summary doesn't exist yet (journal hasn't run for the day, or no chat activity), fall back to the most recent existing \`conversations/summaries/daily/YYYY/MM/DD.md\`.

## Motivation

The journal subsystem already generates a rich daily summary at \`~/<workspace>/conversations/summaries/daily/YYYY/MM/DD.md\` — today's example was 4845 bytes covering art-exhibition research. Currently there's no UI shortcut: the user has to navigate to the FilesView and drill into the date-based directory tree manually. A one-click affordance promotes the daily summary from "buried artifact" to "everyday surface".

### In scope

- New server endpoint \`GET /api/journal/latest-daily\` returning \`{ path: string, isoDate: string } | null\`
  - Searches \`conversations/summaries/daily/\` for the most recent \`YYYY/MM/DD.md\` file that exists
  - \`null\` when no daily summary has ever been generated (fresh install)
- New composable \`src/composables/useLatestDaily.ts\` — fetches the endpoint and navigates to \`/files/<path>\` via vue-router
- New icon-only button in \`src/components/SidebarHeader.vue\` icon cluster (32px square, Material Icons, follows CLAUDE.md chrome-row standard)
- i18n keys in all 8 locales:
  - \`sidebarHeader.todayJournal\` — button tooltip / aria-label (Japanese: "今日のまとめ" / English: "Today's journal" / etc.)
  - \`sidebarHeader.todayJournalNotFound\` — toast/alert text when the endpoint returns null
- Unit test for the server endpoint (golden cases: today exists, today missing → finds yesterday, none exist → null)
- E2E test: render with mocked endpoint, click the button, assert FilesView opens the right path

### Out of scope (deferred)

- A dedicated \`/journal\` or \`/today\` route. Today's path goes through \`/files/<path>\` (existing FilesView md rendering). Adding a custom route is a follow-up if usage justifies it.
- Showing badge / unread indicator on the button when a new daily summary is generated. Could be useful but adds complexity (state tracking, dismissal). Defer until the basic shortcut proves useful.
- A wiki page that auto-includes today's summary. Different UX (read in wiki context vs FilesView). Defer.
- Sidebar / calendar surfacing (option B from chat). Deferred — top-bar icon (option C) ships first, evaluate before adding more entry points.

## UX behaviour (option ii from chat)

- **Today's summary exists** → click navigates to \`/files/conversations/summaries/daily/YYYY/MM/DD.md\` (today's path).
- **Today's missing, prior day exists** → click navigates to the most recent existing daily summary. No special UI; the FilesView title shows the date so the user can see which day they landed on.
- **No daily summary ever generated** → click triggers a non-blocking toast/alert with \`sidebarHeader.todayJournalNotFound\` copy. Button stays clickable (so a user revisiting after journal first runs gets the affordance).

### Server endpoint

\`server/api/routes/journal.ts\` (new file). Walks \`conversations/summaries/daily/\` deepest-first:

1. List year directories, take max
2. List month directories under that year, take max
3. List \`*.md\` files under that month, take max (filter to \`/^\\d{2}\\.md$/\`)
4. Return \`{ path, isoDate }\` (workspace-relative path) or \`null\`

Edge cases to handle in tests:
- Empty \`daily/\` directory
- Year directory with no months
- Month directory with no \`DD.md\` files (e.g. only stray non-numeric files)
- Symlinks / hidden files (\`.DS_Store\`)

### Icon choice

Material Icons \`today\` (calendar-with-checkmark) — semantically obvious. Alternative: \`auto_stories\` (open book), but \`today\` reads as "today's thing" more directly.

### CLAUDE.md compliance

- chrome-row standard: \`h-8 w-8 flex items-center justify-center rounded\`
- icon-cluster group: \`flex gap-0.5\` (existing wrapper around bell/settings already uses this)
- i18n: 8 locales lockstep, English in \`src/lang/en.ts\` first, then translate per locale (do NOT copy English value)
- testid: \`today-journal-btn\` for e2e

## Files to touch

| File | Why |
|---|---|
| \`server/api/routes/journal.ts\` (new) | endpoint |
| \`server/api/routes/index.ts\` (or wherever routes register) | wire up |
| \`src/config/apiRoutes.ts\` | add \`API_ROUTES.journal.latestDaily\` constant |
| \`src/composables/useLatestDaily.ts\` (new) | fetch + navigate |
| \`src/components/SidebarHeader.vue\` | button |
| \`src/lang/{en,ja,zh,ko,es,pt-BR,fr,de}.ts\` (8 files) | i18n keys |
| \`test/api/test_journalRoute.ts\` (new) | server endpoint tests |
| \`e2e/tests/today-journal-button.spec.ts\` (new) | e2e click → navigate |

## Test plan

- [ ] Unit: endpoint with empty daily/ → returns null
- [ ] Unit: endpoint with only \`2026/04/26.md\` → returns that path
- [ ] Unit: endpoint with multiple years → picks the latest
- [ ] Unit: endpoint with junk files (\`.DS_Store\`, non-numeric .md) → ignores them
- [ ] E2E: button visible in top-bar
- [ ] E2E: click with mocked latest-daily endpoint → FilesView opens, URL matches
- [ ] E2E: click with null endpoint → toast appears, no navigation
- [ ] Manual: live server, journal has run today → click opens today's md in FilesView
- [ ] Manual: fresh install → click shows "not found" copy

### PR DESCRIPTION
**Pull request by isamu**
## Summary

Adds a top-bar icon (\`today\`) that opens today's journal daily summary in FilesView. When today's summary hasn't been generated yet, falls back to the most recent existing \`conversations/summaries/daily/YYYY/MM/DD.md\` (option ii from the design conversation). Also tightens spacing between the MulmoClaude logo and icon cluster (~8px tighter).

Closes #876.

## Items to Confirm / Review

- **Banner i18n is broken on the destination** — the \`systemFiles.*\` namespace has been missing from all 8 locale files for a while; clicking the new button lands on a banner showing raw keys (\`systemFiles.journalDaily.title\` etc). Confirmed pre-existing, NOT introduced here. Tracked as a separate follow-up PR (next task) per scope.
- **Toast fallback** — \`useLatestDaily\` uses \`window.alert\` when latest-daily returns null. A proper in-app toast composable doesn't exist yet — alert is the v1 stand-in. Swap when toast infra lands.
- **8-locale lockstep** — \`sidebarHeader.todayJournal\` + \`todayJournalNotFound\` added to en/ja/zh/ko/es/pt-BR/fr/de in that order, translated per locale (no copy-from-English).
- **Spacing tweak** — removed redundant \`mr-1\` on the title span and shrank the home button's right padding \`pr-2 → pr-1\`. Also covers the visual feedback that motivated #876 ("狭くできる？").

## User Prompt

> Journal daily passって全てのユーザで実行されて今日のまとめって全ユーザでつくられるんだよね？そうしたらそこへの(wikiかな？）ショートカットほしいね

Follow-up clarifications (1問1答):

1. 配置 → **C: トップバーのアイコン1個**
2. 当日が無い時 → **ii: 直近の既存 \`daily/*.md\` を探して開く**

Then: \`mulmoclaudeと鍵マークの間って少し狭くできる？\` → applied option B (remove mr-1 + pr-2→pr-1, ~8px tighter).

### Server

- \`server/workspace/journal/latestDaily.ts\` — pure helper, walks \`daily/YYYY/MM/DD.md\` deepest-first with backtrack. Bounded I/O (3 readdir in the happy path). Handles empty newer-month dirs, non-numeric junk files (\`.DS_Store\`, \`9.md\`, \`notes.md\`).
- \`server/api/routes/journal.ts\` — \`GET /api/journal/latest-daily\` thin wrapper, returns \`{ path, isoDate } | null\`.
- Wired into \`server/index.ts\` next to the notifications router.
- Constants: \`API_ROUTES.journal.latestDaily\` in \`src/config/apiRoutes.ts\`.

### Client

- \`src/composables/useLatestDaily.ts\` — fetch + \`router.push\` to \`/files/<path>\`, alert on null.
- \`src/components/SidebarHeader.vue\` — icon button between bell and settings. Chrome-row standard (h-8 w-8, Material Icons \`today\`, testid \`today-journal-btn\`).

### Tests

- 12 unit tests in \`test/journal/test_latestDaily.ts\` (all pass)
- 2 e2e tests in \`e2e/tests/today-journal-button.spec.ts\` (all pass)

## Out of scope (deferred)

- Dedicated \`/journal\` route — uses existing FilesView md path
- Unread / new-summary badge on the button
- Wiki page auto-include of today's summary
- Sidebar / calendar surfacing
- **systemFiles.* i18n namespace fix** — separate PR (next task)

## Test plan

- [x] \`yarn typecheck\` — clean
- [x] \`yarn lint\` — 0 errors (24 pre-existing v-html / similar warnings)
- [x] \`yarn format\` — clean
- [x] \`yarn build\` — ✓ 1155 modules
- [x] \`yarn test\` — 3190+ pass (12 new)
- [x] \`yarn test:e2e --grep today-journal-btn\` — 2/2 pass
- [ ] CI green
- [ ] Manual: live server, click icon → today's md opens in FilesView (fall back to yesterday's if today missing)
- [ ] Manual: spacing tweak verified visually

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: src/composables/useLatestDaily.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  const router = useRouter();
+  const { t } = useI18n();
+  const loading = ref(false);
+
+  async function openLatestDaily(): Promise<void> {
+    if (loading.value) return;
+    loading.value = true;
+    try {
+      const response = await apiGet<LatestDailyResult | null>(API_ROUTES.journal.latestDaily);
+      if (!response.ok || response.data === null) {
```
Comment (isamu) [inline src/composables/useLatestDaily.ts]: `!response.ok` and `response.data === null` are not the same case. Right now a 401/500/network failure will show the same "no journal yet" alert as a genuine empty workspace, which hides real errors and makes the new shortcut misleading. Please split the branches and surface a distinct failure path (and test it).

**Review by isamu — COMMENTED** [review]
One concern blocks this from being a clean LGTM: the new journal shortcut collapses real request failures into the empty-state path, so auth/network/backend problems will tell the user there is no journal yet.

**Comment by isamu** [comment]
CODEX VERDICT: CHANGES REQUESTED
- `src/composables/useLatestDaily.ts`: `!response.ok` is folded into the same alert as a `null` payload, so 401/500/network failures are misreported as "no journal yet". Please separate the error path and add coverage for a non-200 response.

**Comment by isamu** [comment]
CODEX VERDICT: LGTM

I verified the fix is real:
- `src/composables/useLatestDaily.ts` now splits `!response.ok` from the legitimate `null` empty-state branch.
- `todayJournalLoadFailed` is present in all 8 locale files.
- `e2e/tests/today-journal-button.spec.ts` now covers both the `null` case and a `500` failure case, so the distinction is guarded.

No additional correctness, i18n, accessibility, or test-coverage issues stood out in this iteration.

Residual risk: the UX still relies on blocking `window.alert`, but that matches the stated v1 scope until the toast composable exists.
